### DISCUSSION TITLE
Results window hiding query on its 1st apparition

### DISCUSSION THREAD

**Discussion by yansolo82** [dbeaver/dbeaver#13467] [General]
#### Describe the problem you're observing:
The result window is hiding the current query on its 1st apparition, we have go back to the query sheet and scroll down to see the query corresponding to the results

#### Steps to reproduce, if exist:
execute a query which is localized at the end of a query sheet (must have sufficient lines to have scroll)

**Comment by uslss** [comment]
thank you for the report

**Comment by ShadelessFox** [comment]
Hello @yansolo82,

This is how it works in most editors, so I'd say it's standard behavior. 

What's your suggestion?

**Comment by yansolo82** [comment]
it's not the standard behavior in intellij or sqldeveloper (even I think it's a recent behavior in dbeaver) 

after the result window appears you should recenter the edit window so that we can see the line where the cursor is (and maybe a few more lines)

**Comment by ShadelessFox** [comment]
It's standard behavior in IntelliJ. Maybe it's even platform-dependent. 

Imagine you have a small script (e.g. 10 lines) located at the very beginning of your SQL editor and you're trying to resize results panel. After expanding the latter, your script will be hidden somewhere above the SQL editor, because SQL editor is now "glued" to the top of results panel.

**Comment by flambchap** [comment]
That is definitely not a bug. It is a standard behavior on the IntelliJ platform, Eclipse platform, and many other similar applications.
